extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -338,6 +339,17 @@ export class SubscriptionService {
     });
   }
 
+  /**
+   * Logs error and sends message to viewer
+   * @param {string} error
+   */
+  sendAuthTokenErrorToViewer_(error) {
+    this.viewer_.sendMessageAwaitResponse('auth-rejected', dict({
+      'reason': error,
+    }));
+    user().error(error);
```

Comment:
With `user().error()` you need to also specify TAG and message. E.g. 
```
user().error(TAG, 'rejected authorization: ', error)
```

Reply:
Done